config.js

```diff
@@ -79,30 +57,17 @@ exports.updateFromArgv = function(argv) {
     if (argv.logfile && typeof argv.logfile === "boolean") {
         exports.logfile = `gtp2ogs_logfile_${new Date().toISOString()}`;
     }
-    for (const k of ["timeout", "startupbuffer"]) {
-        if (argv[k]) {
-            // Convert some times to microseconds once here so
-            // we don't need to do it each time it is used later.
-            exports[k] = argv[k] * 1000;
-        }
-    }
+    exportMicroseconds("timeout", argv);
+    exportMicroseconds("startupbuffer", argv);
+
     if (argv.beta) {
         exports.host = 'beta.online-go.com';
     }
 
-    // Setting minimum handicap higher than -1 has the consequence of disabling
-    // automatic handicap (notification.handicap === -1).
-    //
-    if (argv.minhandicap > -1) {
-        exports.noautohandicap = true;
-    }
-    if (argv.minhandicapranked > -1) {
-        exports.noautohandicapranked = true;
-    }
-    if (argv.minhandicapunranked > -1) {
-        exports.noautohandicapunranked = true;
-    }
-
+    exportNoAutoHandicapIfMinHandicapIsPositive("", argv);
+    exportNoAutoHandicapIfMinHandicapIsPositive("ranked", argv);
+    exportNoAutoHandicapIfMinHandicapIsPositive("unranked", argv);
```

Comment:
Wrap these 3 in a function.

Reply:
fixed :100:

Reply:
Did you?

Reply:
oh i did it only for exportInitialConfig() at line 13
sure i can do it here too 🛰️

Reply:
fixed
and it does look a lot nicer :100: :open_mouth: :100: :trophy: